frontend/src/i18n/translations/en-US.ts

```diff
@@ -468,6 +468,9 @@ export default {
   "admin.config.share.max-expiration": "Max expiration",
   "admin.config.share.max-expiration.description":
     "Maximum share expiration in hours. Set to 0 to allow unlimited expiration.",
+  "admin.config.share.share-id-length": "Default share ID length",
+  "admin.config.share.share-id-length.description":
+    "Default length for the generated ID of a share. This value is also used to generates links for reverse shares. A value under 8 is not considered secure.",
```

Comment:
```suggestion
    "Default length for the generated ID of a share. This value is also used to generate links for reverse shares. A value below 8 is not considered secure.",
```

Reply:
Oops, sorry for the typos 🫣 I just pushed a revision.